### PR TITLE
Add more debugging to high density runs to help with debugging

### PR DESCRIPTION
**Pull request by gmarek**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1106)

### CODE REVIEW

File: jobs/ci-kubernetes-kubemark-high-density-100-gce.sh [hunk 1]
```diff
@@ -45,10 +45,18 @@ export MASTER_SIZE="n1-standard-2"
...
 export KUBEMARK_NUM_NODES="600"
+
 # The kubemark scripts build a Docker image
 export JENKINS_ENABLE_DOCKER_IN_DOCKER="y"
 export KUBE_NODE_OS_DISTRIBUTION="gci"
 
+# TODO: remove after we stabilize performance of this suite.
+export KUBEMARK_MASTER_ROOT_DISK_SIZE="100GB"
+export MASTER_DISK_SIZE="100GB"
+export APISERVER_TEST_ARGS="--v=2"
```
Comment (wojtek-t) [inline jobs/ci-kubernetes-kubemark-high-density-100-gce.sh]: this is default.

File: jobs/ci-kubernetes-kubemark-high-density-100-gce.sh [hunk 2]
```diff
@@ -45,10 +45,18 @@ export MASTER_SIZE="n1-standard-2"
...
+export SCHEDULER_TEST_ARGS="--v=4"
+export CONTROLLER_MANAGER_TEST_ARGS="--v=2"
```
Comment (wojtek-t) [inline jobs/ci-kubernetes-kubemark-high-density-100-gce.sh]: This one too.

File: jobs/ci-kubernetes-kubemark-high-density-100-gce.sh (hunk 1 above)
Comment (gmarek) [inline jobs/ci-kubernetes-kubemark-high-density-100-gce.sh]: Done.

File: jobs/ci-kubernetes-kubemark-high-density-100-gce.sh (hunk 2 above)
Comment (gmarek) [inline jobs/ci-kubernetes-kubemark-high-density-100-gce.sh]: Done.

**Pull request merged**